tst/integration/test_json_basic.py

```diff
@@ -1887,7 +1887,7 @@ def test_json_strappend_command_legacy_and_jsonpath_wildcard(self):
             assert exp_new_str == client.execute_command(
                 'JSON.GET', key, path).decode()
 
-    def test_json_objectlen_command(self):
+    def test_json_object_len_command(self):
```

Comment:
the actual things are `objlen` or `objkeys`, so I don't see any particular reason to invent additional new words just for these tests...